cmd/bosun/sched/template.go

```diff
@@ -367,3 +367,19 @@ func (c *Context) HTTPGet(u string) string {
 	}
 	return string(body)
 }
+
+func (c *Context) HTTPPost(u, t, d string) string {
+	resp, err := http.Post(u, t, bytes.NewBufferString(d))
+	if err != nil {
+		return err.Error()
+	}
+	if resp.StatusCode >= 300 {
+		return fmt.Sprintf("%v: returned %v", u, resp.Status)
+	}
+	defer resp.Body.Close()
```

Comment:
This needs to be before the previous if block. Otherwise the body will remain open if the response is >= 300 (and the body can contain data in that case). This also needs to be fixed in the Get handler above.